### ISSUE TITLE
FirebaseAuth is undefined error

### ISSUE THREAD

**Issue by jasan-s** [firebase/firebaseui-web-react#1]
> React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in.

`import { FirebaseAuth } from 'react-firebaseui';`

` <FirebaseAuth uiConfig={this.uiConfig}  firebaseAuth={firebase.auth()}/>`

I'm using this with `Create React App` build APP

**Comment by nicolasgarnier** [comment]
Hey @jasan-s I kind of messed up with creating the release files. You canuse this in the mean time:

```js
import { FirebaseAuth } from 'react-firebaseui/dist';
```

or

```js
import FirebaseAuth from 'react-firebaseui/dist/FirebaseAuth';
```

I'm fixing this in the next version.

**Comment by nicolasgarnier** [comment]
All fixed in v 1.0.5. You can do as instructed from the README:

```js
import { FirebaseAuth } from 'react-firebaseui';
// or
import FirebaseAuth from 'react-firebaseui/FirebaseAuth';
```
